### DISCUSSION TITLE
plot has a farmer public key that is not in the farmer's pk list

### DISCUSSION THREAD

**Discussion by fabitom** [Chia-Network/chia-blockchain#3787] [General]
``plot has a farmer public key that is not in the farmer's pk list
``
Should I ignore that warning ? I've some plots generated on other machine with main -f farmer_key -p pool_key.
Or should I add my second farmer keys to main ?

**Comment by cryptoshaman007** [comment]
You can farm with those plots?
I´m getting same error on my plots on one computer today, the others are fine.
Its not farming these with errors

How to fix?? Not found anything, tried deleting keys and reentering, reinstalling all. Checked and the keys are right, but don´t work.
Was all working fine!! 
Did a chkdsk on one drive that had a bad cluster and one plot only, nothing more.

**Reply by fabitom** [reply]
The problem was with moving farm. Just moving config/cert files was wrong. You need to use recovery from 24 words.

**Reply by LDizon** [reply]
Did you just add the old farmer keys to the new farmer? Does it overwrite the new keys or just add more?

**Comment by sadegh1010** [comment]
I did trouble mistake. I reinstall windows on main machine. and now my farmer machine log shows  "plot has a farmer public key that is not in the farmer's pk list".
I've added 24 seeds.
i don't have previous farmer public key!

**Comment by BrandtH22** [comment]
@fabitom , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
